vespa-cloud/document-processing/README.md

```diff
@@ -21,115 +21,139 @@ and the asynchronous [Document API](https://docs.vespa.ai/en/document-api-guide.
 
 Flow:
 1. Feed album document with the _music_ schema
-1. Look up in the _lyrics_ schema if album with given ID has lyrics stored
-1. Store album with lyrics in the _music_ schema
+2. Look up in the _lyrics_ schema if album with given ID has lyrics stored
+3. Store album with lyrics in the _music_ schema
```

Comment:
Using only `1.` allows the Markdown renderer to generate the sequence.

Reply:
yup :-) auto-corrected by WebStorm